### ISSUE TITLE
Toml support in UserFunction_ class

### ISSUE THREAD

**Issue by vickysharma0812** [vickysharma0812/easifem-classes#180]
Currently Toml config for UserFunction, which is given below, does not contain the name field. Please include the name field.

```toml 
[userFunction]
returnType = "Scalar"
# returnType = "Vector"
# returnType = "Matrix"
argType = "Space"
# argType = "Constant"
# argType = "Space"
# argType = "Time"
# argType = "SpaceTime"
numArgs = 3
numReturns = 1
luaScript = "./hello.lua"
luaFunctionName="hello"
scalarValue = 1.0 
vectorValue = [1.0, 2.0, 3.0]
matrixValue = [[1.0, 2.0, 3.0], [4.0, 5.0, 6.0], [7.0, 8.0, 9.0]]
```

**Comment by vickysharma0812** [comment]
Fixed in #189

**Comment by vickysharma0812** [comment]
Fixed in #189
